### DISCUSSION TITLE
npu环境下推理速度极慢

### DISCUSSION THREAD

**Discussion by liujiachang** [PaddlePaddle/PaddleOCR#13733] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

在昇腾910B上运行paddleocr代码，npu推理速度远远大于cpu，同一图片多次推理速度变正常。

### 🏃‍♂️ Environment (运行环境)

操作系统ubuntu20.04
python=3.10
paddlepaddle=3.0.0b1

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

直接运行仓库代码即可，模型使用的ch_PP-OCRv4_det_infer/ch_PP-OCRv4_rec_infer

**Comment by ACFFF** [comment]
遇到了相同的问题，请问现在有解决方法了没？
